### PR TITLE
docs: use "containers-" prefix and add containers-mounts.conf

### PR DESCRIPTION
**Pull request by vrothberg**
- docs: use "containers-" prefix for registries and storage
 - docs: add containers-mounts.conf(5)

### CODE REVIEW

**Comment by vrothberg** [comment]
@TomSweeneyRedHat @rhatdan PTAL

**Comment by vrothberg** [comment]
We need similar changes for Buildah and Skopeo (Cc: @mtrmac @runcom). I already have a PR open against CRI-O.

File: docs/containers-mounts.conf.5 [hunk 1]
```diff
@@ -0,0 +1,16 @@
+% containers-mounts.conf(5) configuration files for default mounts in containers
+
+# NAME
+containers-mounts.conf - configuration files for default mounts in containers
+
+# DESCRIPTION
+The mounts.conf files specify volume mount directories that are automatically mounted inside containers. Container processes can then use this content. Usually these directories are used for passing secrets or credentials required by the package software to access remote package repositories. Note that for security reasons, tools adhering to the mounts.conf are expected to copy the contents istead of bindmounting the paths from the host.
```
Comment (TomSweeneyRedHat) [inline docs/containers-mounts.conf.5]: "mounts.conf files" to "mounts.conf file"

Comment (giuseppe) [inline docs/containers-mounts.conf.5]: typo: istead -> instead

**Comment by vrothberg** [comment]
> I know a lot of this was cut/paste, but if you could considert changing it, I'd appreciate it.

With pleasure! Thanks for the reviews, @TomSweeneyRedHat and @giuseppe !

**Comment by vrothberg** [comment]
@TomSweeneyRedHat, @giuseppe, I addressed your comments and also renamed the file to `*.5.md` to make it adhere to `make docs`.

**Comment by rhatdan** [comment]
LGTM
@rh-atomic-bot r+

**Pull request closed without merging**